doc/api/test.md

```diff
@@ -303,6 +303,9 @@ added: v18.0.0
   * `todo` {boolean|string} If truthy, the test marked as `TODO`. If a string
     is provided, that string is displayed in the test results as the reason why
     the test is `TODO`. **Default:** `false`.
+  * `timeout` {number} A number of milliseconds the test will fail after.
```

Comment:
Add it to the YAML above (below the method name) since it was added after 18.0 (add a REPLACEME, follow the blame for sytntax)